### ISSUE TITLE
Quest: The Demon Scarred Cloak

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#77]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Mulgore / Item: Demon Scarred Cloak / Quest: The Demon Scarred Cloak
Link: https://www.wowhead.com/wotlk/quest=770/the-demon-scarred-cloak

Description: the voice-over suddenly silences at the end of the playback. 

Progress: no playback.

Completion: playback ok.

**Comment by carevass** [comment]
This should be fixed in AI_VoiceOverData_VanillaExtra_v1.5.
